common/scripted_effects/hfe_effect_primitive.txt

```diff
@@ -370,44 +370,44 @@ hfe_effect_primitive_set_early_space_age = {
 			file = new_dawn.dds
 		}
 		colors = {
-			grey
-			dark_grey
-			null
-			null
+			"grey"
+			"purple"
+			"dark_grey"
+			"black"
 		}
 	}
 }
 
 hfe_effect_primitive_set_random_age = {
 	random_list = {
-		10 = {
-			hfe_effect_primitive_set_stone_age = yes
-		}
-		10 = {
-			hfe_effect_primitive_set_bronze_age = yes
-		}
-		10 = {
-			hfe_effect_primitive_set_iron_age = yes
-		}
-		10 = {
-			hfe_effect_primitive_set_late_medieval_age = yes
-		}
-		10 = {
+#		10 = {
```

Comment:
Why?

Reply:
Using technology to erase all knowledge of FTL is one thing. Turning a fully sapient FTL nation into a race of cavemen is another. It's too weird.

Reply:
I see nothing weird here and this effect used (or not but will be) in another events.